lib/bitbucket_rest_api/repos/services.rb

```diff
@@ -12,12 +12,19 @@ class Repos::Services < API
     #  bitbucket.repos.services.list 'user-name', 'repo-name'
     #  bitbucket.repos.services.list 'user-name', 'repo-name' { |service| ... }
     #
-    def list(user_name, repo_name, params={})
-      _update_user_repo_params(user_name, repo_name)
+    def list(user_name_or_project_key, repo_name, params={})
+      _update_user_repo_params(user_name_or_project_key, repo_name)
       _validate_user_repo_params(user, repo) unless user? && repo?
       normalize! params
 
-      response = get_request("/1.0/repositories/#{user}/#{repo.downcase}/services", params)
+      url = if BitBucket.options[:bitbucket_server]
+              "/1.0/projects/#{user_name_or_project_key}/repos/#{repo.downcase}/settings/hooks"
+            else
+              "/1.0/repositories/#{user}/#{repo.downcase}/services"
+            end
+
+      response = get_request(url, params)
+
```

Comment:
You're right. It seems to weird to me that this would exist here and then we would also have something in the `BitBucket::Repos::Webhooks`. I think this is fine though!